dist/index.html

```diff
@@ -6,91 +6,69 @@
 <body>
 
   <div class="container">
-    
     <div class="container_screen">
       <input type="form" id="inputScreen" class="screen display" readonly="readonly">
       <input type="form" id="outputScreen" class="outputscreen display" readonly="readonly"> 
     </div>
-
     <div class="contrainer_button">
       <button class="button" id="AC">
           AC
       </button>
-
       <button class="button operation" id="modulus">
         <img id="modulusIcon">
       </button>
-
       <button class="button operation" id="division">
         <img id="divisionIcon">
       </button>
-
       <button class="button number" id="7">
         7
       </button>
-
       <button class="button number" id="8">
         8
       </button>
-
       <button class="button number" id="9">
         9
       </button>
-
       <button class="button operation" id="multiplication">
         <img id="multiplicationIcon">
       </button>
-
       <button class="button number" id="4">
         4
       </button>
-
       <button class="button number" id="5">
         5
       </button>
-
       <button class="button number" id="6">
         6
       </button>
-
       <button class="button operation" id="subtraction">
         <img id="subtractionIcon">
       </button>
-
       <button class="button number" id="1">
         1
       </button>
-
       <button class="button number" id="2">
         2
       </button>
-
       <button class="button number" id="3">
         3
       </button>
-
       <button class="button operation" id="addition">
         <img id="additionIcon">
       </button>
-
       <button class="button number" id="0">
         0
       </button>
-
       <button class="button operation" id="point">
         <img id="pointIcon">
       </button>
-
       <button class="button" id="delete">
         <img id="deleteIcon">
       </button>
-
       <button class="button operation" id="equals">
         <img id="equalIcon">
       </button>
-
     </div>
-
   </div>
 
```

Comment:
Remove Line gap

Reply:
Removed the line gap and I have committed the change